src/main/java/pl/javastart/task/VehicleTools.java

```diff
@@ -0,0 +1,93 @@
+package pl.javastart.task;
+
+import java.io.*;
+import java.util.LinkedList;
+import java.util.Queue;
+import java.util.Scanner;
+
+public class VehicleTools {
+
+    private static final int EXIT = 0;
+    private static final int ADD_TO_QUEUE = 1;
+    private static final int TAKE_FOR_SERVICE = 2;
+    Scanner scanner = new Scanner(System.in);
+
+    void run() throws IOException {
+        String fileName = "vehicles.txt";
+        Queue<Vehicle> vehicles = new LinkedList<>();
+        readFileAndAddToQueue(fileName, vehicles);
```

Comment:
Nie jest to błąd, ale preferowałbym metody, które zwracają wynik, niż takie, które przyjmują argument i go modyfikują. Czyli w tym przypadku:
```
Queue<Vehicle> vehicles = readFile(fileName);
```
Zalety to krótszy kod i krótsza lista parametrów metody